### PR TITLE
New Tooltip Feature added

### PR DESCRIPTION
**Pull request by amansoni7477030**
1. Added tooltip feature to enhance user interaction:
Users can now control tooltip visibility using the showTooltip parameter (true for visible, false for hidden). Additionally, they can customize the tooltip's background and text colors, as well as the message itself with the following parameters:
```
showTooltip: true,
tooltipMessage: 'Hi There 👋!',
tooltipBackgroundColor: 'black',
tooltipTextColor: 'white',
```
Screenshot from 2024-06-13 12-34-31

Note: This feature request was previously mentioned in enhancement #76 and discussed at https://github.com/FlowiseAI/Flowise/discussions/1453

2. Fixed issue where the send button would go below the text window when the text input was long:

Screenshot from 2024-06-13 10-42-09

3. Updated the title of the new chat and added the title to the close button.

### CODE REVIEW

File: src/components/inputs/textInput/components/TextInput.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ export const TextInput = (props: Props) => {
 
   return (
     <div
-      class="w-full h-auto max-h-[128px] min-h-[56px] flex flex-col items-end justify-between chatbot-input border border-[#eeeeee]"
+      class="w-full h-auto max-h-[192px] min-h-[56px] flex flex-col items-end justify-between chatbot-input border border-[#eeeeee]"
```
Comment (HenryHengZJ) [inline src/components/inputs/textInput/components/TextInput.tsx:86]: why the change from 128px to 192px? Do you have a screenshot of the difference? I couldnt find it

**Comment by HenryHengZJ** [comment]
hmm very strange, my tool tip emoji not showing:
image

File: src/components/inputs/textInput/components/TextInput.tsx (hunk 1 above)
Comment (HenryHengZJ) [reply src/components/inputs/textInput/components/TextInput.tsx:86]: oh thats because `Fixed issue where the send button would go below the text window when the text input was long:`

Comment (HenryHengZJ) [reply src/components/inputs/textInput/components/TextInput.tsx:86]: before:
image

Comment (HenryHengZJ) [reply src/components/inputs/textInput/components/TextInput.tsx:86]: after:
image

**Comment by HenryHengZJ** [comment]
can we also allow user to specify the font size? if not we will take the default 16 font size

**Comment by amansoni7477030** [comment]
> can we also allow user to specify the font size? if not we will take the default 16 font size

Hii @HenryHengZJ tooltip fontsize parameter added now check latest update.

**Comment by amansoni7477030** [comment]
> hmm very strange, my tool tip emoji not showing: image

@HenryHengZJ its perfectly working fine and i am  not facing any issue.
check attached image 

Screenshot from 2024-06-14 17-43-06

File: src/components/inputs/textInput/components/TextInput.tsx (hunk 1 above)
Comment (amansoni7477030) [reply src/components/inputs/textInput/components/TextInput.tsx:86]: yes that's the issue i solved in this

**Comment by toi500** [comment]
Just tested and it works with icons for me.

image

**Comment by toi500** [comment]
Can you guys incorporate this simple fix into this PR (misconfiguration with the left and right top borders on mobile)?

via `rounded-lg` 

image

or 

```css
@media (max-width: 640px) {
    div[part=bot] {
        border-radius: 0;
    }
}
```

**Comment by amansoni7477030** [comment]
> Can you guys incorporate this simple fix into this PR (misconfiguration with the left and right top borders on mobile)?
> 
> via `rounded-lg`
> 
> image
> or
> 
> ```css
> @media (max-width: 640px) {
>     div[part=bot] {
>         border-radius: 0;
>     }
> }
> ```

@toi500 sure

**Review by HenryHengZJ — APPROVED** [review]
some conflict on the `dist/web.js`, otherwise looks good! thank you!!

**Comment by amansoni7477030** [comment]
Hii @HenryHengZJ i resolved the conflict can you please review and merge it to main

**Review by HenryHengZJ — APPROVED** [review]
